src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -137,12 +134,13 @@ QByteArray AuthenticationInAppListener::generateAuthPw() const {
   return hash.result(32, "identity.mozilla.com/picl/v1/authPW");
 }
 
+void AuthenticationInAppListener::setPassword(const QString& password) {
+  m_password = password;
```

Comment:
As a nitpicky thing, I would store the authPW instead of the user's unhashed password here just to make it a little harder to grep for passwords out of a coredump. Stealing someone's authPW could compromise their FxA account, but stealing the unhashed password could compromise a wider set of services.

Reply:
Good point! done.